acceptance-tests/src/main/kotlin/org/ostelco/at/common/Products.kt

```diff
@@ -32,6 +32,24 @@ val expectedPlanProduct: Product = Product()
         )
         .presentation(emptyMap<String, String>())
 
+val expectedPlanProductUS: Product = Product()
+        .sku("PLAN_10USD_DAY")
+        .price(Price().amount(10_00).currency("USD"))
+        .properties(
+                mapOf(
+                        "productClass" to "SUBSCRIPTION",
+                        "segmentIds" to "country-us"
+                )
+        )
+        .payment(
+                mapOf(
+                        "type" to "SUBSCRIPTION",
+                        "label" to "Daily subscription plan",
+                        "taxRegionId" to "us"
+                )
+        )
```

Comment:
Here, the payment type is set correctly to "SUBSCRIPTION", but the product class should be MEMBERSHIP.